docs/corpora_schema.md

```diff
@@ -69,44 +84,44 @@ corpora_encoding_version|Follows [SemVer](https://semver.org/) conventions
 
 ### Integration Metadata
 
-To support data integration, each cell must have the following metadata:
+To support data integration, each cell MUST have the following metadata:
 
 **Field name**|**Constraints**
 :--|:--
-tissue|string (see note below regarding cell culture and organoids)
+tissue|string. This field SHOULD be appended with " (cell culture)" or " (organoid)" if appropriate.
 assay|string
 disease|string
 cell\_type|string
 sex|"male", "female", "mixed", "unknown", or "other"
 ethnicity|string, "na" if non-human, "unknown" if not available
 development\_stage|string, "unknown" if not available
 
-The `tissue` field must be appended with " (cell culture)" or " (organoid)" if appropriate. Also, if the source of cells is cell culture or organiod, the
-`cell_type` field can be left empty.
-In addition to these free text fields (except sex), each cell must also have ontology values:
+In addition to these free text fields (except sex), each cell MUST also have ontology values:
 
 **Field name**|**Constraints**
 :--|:--
-tissue\_ontology\_term\_id|UBERON term
+tissue\_ontology\_term\_id|UBERON term. This field SHOULD be appended with " (cell culture)" or " (organoid)" if appropriate.
 assay\_ontology\_term\_id|EFO term
 disease\_ontology\_term\_id|MONDO term or [PATO:0000461](http://bioportal.bioontology.org/ontologies/PATO?p=classes&conceptid=PATO%3A0000461)
 cell\_type\_ontology\_term\_id|CL term
 ethnicity\_ontology\_term\_id|HANCESTRO term, "na" if non-human
 development\_stage\_ontology\_term\_id|HsapDv term if human, child of EFO:0000399 otherwise
 
-The `tissue_ontology_term_id` field must be appended with " (cell culture)" or " (organoid)" if appropriate.
+cellxgene requires ontology terms to enable search, comparison, and integration of data. When no appropriate ontology value is available, then the most
+precise accurate term MUST be used. For example if the `cell_type` field describes a relay interneuron, but the most specific available term in the CL
```

Comment:
What about the cases that there is no cell type available in the ontology? We could suggest the root of the ontology tree, or maybe one of the first children:

-- cell
----- native cell (A cell that is found in a natural setting)
----- cell _in vitro_
----- abnormal cell

A dataset of the human embryo exemplifies the existence of "unassignable" cell types
[https://docs.google.com/spreadsheets/d/14NfyiUWGOzgcRg7nUDiLNO9rMHUef6-gM1f7rlR5ymE/edit#gid=1775586196](https://docs.google.com/spreadsheets/d/14NfyiUWGOzgcRg7nUDiLNO9rMHUef6-gM1f7rlR5ymE/edit#gid=1775586196)

Reply:
Great suggestion, thanks. Please review 7d079fa for how I integrated.